lore/settings.py

```diff
@@ -237,7 +237,9 @@ def get_var(name, default):
 if LORE_USE_S3:
     DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
 else:
-    DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
+    # by default use django.core.files.storage.FileSystemStorage with
+    # overwrite feature
+    DEFAULT_FILE_STORAGE = 'storages.backends.overwrite.OverwriteStorage'
```

Comment:
After thinking more about it, I don't think there is a problem with S3, since the problem was related to the local storage, but we may want to make some tests